test/inspect.test.js

```diff
@@ -260,6 +297,52 @@ test('inspect', (t) => {
     });
 });
 
+test('inspect --only-provenance', (t) => {
+  return Promise.resolve()
```

Comment:
async/await maybe will be consistent ?

Reply:
I'd like to upgrade the whole file to typescript but let's do that later.

Reply:
For now, extracted this test into a separate file.